README.md

```diff
@@ -149,6 +149,30 @@ You can update multiple docs versions at the same time because each directory in
 
 Example: When you change any file in `versioned_docs/version-0.7.0/`, it will only affect the docs for version `0.7.0`.
 
+## Configs
```

Comment:
Instead of copying the docs, maybe better to provide a link to the `hudi-utils/README.md` in this section?